runbotics/runbotics-scheduler/src/scheduler-database/tenant/tenant-invite-code.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({ schema: 'public' })
+export class TenantInviteCode {
+    @PrimaryGeneratedColumn('uuid', { name: 'invite_id' })
+    inviteId: string;
+
+    @Column({ type: 'uuid', name: 'tenant_id' })
+    tenantId: string;
+
+    @Column({ type: 'timestamp without time zone', name: 'creation_date', nullable: false })
+    creationDate: Date;
+
+    @Column({ type: 'timestamp without time zone', name: 'expiration_date', nullable: false })
+    expirationDate: Date;
```

Comment:
nullable is redundant - it's false by default

Reply:
and should be createdDateColumn and UpdatedDateColumn - I've left additional comment below